### ISSUE TITLE
Question About Normal Map Resolution

### ISSUE THREAD

**Issue by CXCxyz** [wlydlut/UHDDIP#2]
Hi, are the normal maps in your project the same ultra-high resolution as the original images, or are they processed at a different resolution?

**Comment by wlydlut** [comment]
The normal maps are the same ultra-high resolution as the original images.

---- Replied Message ----
| From | ***@***.***> |
| Date | 02/15/2025 18:46 |
| To | wlydlut/UHDDIP ***@***.***> |
| Cc | Subscribed ***@***.***> |
| Subject | [wlydlut/UHDDIP] Question About Normal Map Resolution (Issue #2) |

Hi, are the normal maps in your project the same ultra-high resolution as the original images, or are they processed at a different resolution?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

CXCxyz created an issue (wlydlut/UHDDIP#2)

Hi, are the normal maps in your project the same ultra-high resolution as the original images, or are they processed at a different resolution?

**Comment by 18018982653** [comment]
How should the validation set be set? Is it partitioned from the training set? What should the path to the validation set be?

**Comment by 18018982653** [comment]
Is the `normal prior` obtained by executing the code in "https://github.com/EPFL-VILAB/omnidata" and then running `demo.py`, specifically the line `# Surface normal estimation model model_normal = torch.hub.load('alexsax/omnidata_models', 'surface_normal_dpt_hybrid_384')`?

**Comment by wlydlut** [comment]
To reduce training time, the validation set consists of three images (e.g., one input image, one GT image, one normal image) , where randomly selected from the testing set.


Place one image in each of the following paths:
dataroot_gt: /dataset/UHD/val/gt
dataroot_lq: /dataset/UHD/val/input
dataroot_normal: /dataset/UHD/val/normal






---- Replied Message ----
| From | ***@***.***> |
| Date | 11/21/2025 20:01 |
| To | ***@***.***> |
| Cc | liyan ***@***.***>,
***@***.***> |
| Subject | Re: [wlydlut/UHDDIP] Question About Normal Map Resolution (Issue #2) |
18018982653 left a comment (wlydlut/UHDDIP#2)

How should the validation set be set? Is it partitioned from the training set? What should the path to the validation set be?

**Comment by wlydlut** [comment]
The normal prior is obtained by https://github.com/EPFL-VILAB/omnidata/tree/main/omnidata_tools/torch/demo.py


Then, the normal prior is resized the same ultra-high resolution as the original images by "transforms.Resize()".






---- Replied Message ----
| From | ***@***.***> |
| Date | 11/21/2025 20:56 |
| To | ***@***.***> |
| Cc | liyan ***@***.***>,
***@***.***> |
| Subject | Re: [wlydlut/UHDDIP] Question About Normal Map Resolution (Issue #2) |
18018982653 left a comment (wlydlut/UHDDIP#2)

Is the normal prior obtained by executing the code in "https://github.com/EPFL-VILAB/omnidata" and then running demo.py, specifically the line # Surface normal estimation model model_normal = torch.hub.load('alexsax/omnidata_models', 'surface_normal_dpt_hybrid_384')?

**Comment by 18018982653** [comment]
Hello, I followed your instructions: The normal prior is obtained using https://github.com/EPFL-VILAB/omnidata/tree/main/omnidata_tools/torch/demo.py

Then, the normal prior is resized to the same ultra-high resolution as the original images using "transforms.Resize()". After execution, I found that the generated normal image is not identical to the original 4K input image; it has lost surrounding pixels, making it appear enlarged. How can I resolve this?

**Comment by 18018982653** [comment]
To reduce computational cost, I downsampled the 4k input to 384*384, then input it into the model to obtain the normal figure, and then upsampled it back to 4k. The output then matched the previous result. Thank you, sorry to bother you.
